packages/dashboard/src/app/views/myStories/karma/myStories.karma.js

```diff
@@ -127,7 +127,7 @@ describe('CUJ: Creator can view their stories in grid view', () => {
 
       await fixture.events.click(exploreTemplatesMenuItem);
       const templatesGridEl = await waitFor(() =>
-        fixture.screen.queryByText('Viewing all templates')
+        fixture.screen.findByText('Viewing all templates')
```

Comment:
From the [Testing Library docs](https://testing-library.com/docs/dom-testing-library/api-async#findby-queries):

> `findBy` methods are a combination of `getBy` queries and `waitFor`. They accept the `waitFor` options as the last argument

Reply:
Oh thats right! Thanks!